tests/low-ram/low-ram.sh

```diff
@@ -4,22 +4,46 @@ set -euo pipefail
 
 cd "$(dirname "${BASH_SOURCE[0]}")"
 
-docker build ../../ --tag=qdrant_consensus
+declare DOCKER_IMAGE_NAME=qdrant-recovery
 
-if [[ ! -e storage ]]
-then
+docker build ../../ --tag=$DOCKER_IMAGE_NAME
+
+if [[ ! -e storage ]]; then
     git lfs pull
     tar -xf storage.tar.xz
 fi
 
+declare OOM_CONTAINER_NAME=qdrant-oom
+
+docker rm -f ${OOM_CONTAINER_NAME} || true
+
+docker run \
+    -m 128m \
+    -p 127.0.0.1:6333:6333 \
+    -p 127.0.0.1:6334:6334 \
+    -v $PWD/storage:/qdrant/storage \
+    -e QDRANT__STORAGE__HANDLE_COLLECTION_LOAD_ERRORS=true \
+    --name ${OOM_CONTAINER_NAME} \
+    $DOCKER_IMAGE_NAME || true
```

Comment:
I don't think we want to `|| true` here? Failing to start the container should fail the test.

Reply:
this part actually checks the container failed